server/controllers/events/events_controller_e2e_test.go

```diff
@@ -806,10 +806,12 @@ func setupE2E(t *testing.T, repoFixtureDir string, userConfig *server.UserConfig
 	// swapping out version cache to something that always returns local contest
 	// binary
 	conftextExec.VersionCache = conftestCache
-
+	// e2e test will be using legacy mode for now
```

Comment:
hmm we should probably not launch without this on our e2e tests though.

Reply:
The difference between legacy and new version is that we now have a dependency on the GH client creator. I would still need to mock that part out, but will add it.